kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -645,6 +645,23 @@ object KyuubiConf {
       .stringConf
       .createWithDefault("uid")
 
+  val AUTHENTICATION_LDAP_PASSWORD: OptionalConfigEntry[String] =
+    buildConf("kyuubi.authentication.ldap.bindnpw")
+      .doc("The password of the LDAP bind user " +
+        "is used to search for the full name of the authenticated user.")
+      .version("1.6.0")
+      .stringConf
+      .createOptional
+
+  val AUTHENTICATION_LDAP_ATTRIBUTES: ConfigEntry[Seq[String]] =
+    buildConf("kyuubi.authentication.ldap.attrs")
+      .doc("Specifies part of the search as an attribute returned by LDAP. " +
+        "For example, mail,name.")
+      .version("1.6.0")
+      .stringConf
+      .toSequence()
+      .createWithDefault(Seq("mail"))
```

Comment:
why mail is the default value?

Reply:
val searchFilter = String.format("(%s=%s)", "mail", mail)
nameEnuResults = ctx.search(baseDn, searchFilter, sc)
Set a default value that can override the case where kyuubi.authentication.ldap.attrs does not exist.